tempesta_fw/http.c

```diff
@@ -259,27 +311,46 @@ tfw_http_send_resp(TfwHttpReq *req, TfwStr *msg, const TfwStr *date)
 		msg->len += crlf->len - crlf_len;
 	}
 
-	if (!(hmresp = tfw_http_msg_alloc_err_resp()))
-		return -ENOMEM;
+	if (!(hmresp = tfw_http_msg_alloc_err_resp())) {
+		TFW_DBG2("%s: Response message allocation error: conn=[%p]\n",
```

Comment:
I don't think, that we need so precise debugging for memory errors. It seems reasonable to put just single `TFW_DBG2` after `err_setup` label.

Reply:
Corrected.